device.go

```diff
@@ -211,6 +215,24 @@ func (d *Device) GetStringDescriptor(descIndex int) (string, error) {
 	return libusb.getStringDesc(d.handle, descIndex)
 }
 
+// Manufacturer returns the device's manufacturer name as string.
```

Comment:
I think you can drop "as string", as it's explicit in the function signature.

Reply:
done